wgpu/src/util/init.rs

```diff
@@ -86,3 +86,21 @@ pub async fn initialize_adapter_from_env_or_default(
         }
     }
 }
+
+/// Choose which DX12 shader compiler to use from the environment variable WGPU_DX12_COMPILER
+pub fn dx12_shader_compiler_from_env() -> Option<wgt::Dx12Compiler> {
```

Comment:
This is nice for testing, and could probably be used in the future for running tests on FXC and DXC. I didn't bother trying to figure out how to parse paths from the environment variable so it only checks locally. (or wherever `libloading` checks, I'm not 100% sure how it works).

Reply:
It just defers to the system style - for this kind of thing "just put it in your working dir" is perfectly reasonable - so you made the right call not bothering. The existance of this env var should be documented in the readme though.